pkg/apis/kubeone/types.go

```diff
@@ -205,6 +207,13 @@ type Features struct {
 	OpenIDConnect     *OpenIDConnect     `json:"openidConnect"`
 }
 
+// SystemPackages controls configurations of APT/YUM
+type SystemPackages struct {
```

Comment:
What if we call this `SystemConfiguration` or maybe even `HostConfiguration` (so it's matching `hosts`), so we can easily re-use this field for other purposes without having to bump the API or add new fields?

Reply:
This API is only about packages, not system or host in general.